sdk/servicebus/service-bus/test/stress/stress_lockRenewal.ts

```diff
@@ -0,0 +1,107 @@
+/*
+Test Scenario summary:
+- Creates a single sender and a single receiver on a queue.
+- Receives a message and holds onto it for the duration of test. AutoLockRenewal is enabled and set to the test duration.
+
+The test assumes no other process is working with the queues defined in here,
+but the queues must be empty and use default configurations before running the test.
+
+For running this test, connection string of the Service Bus namespace and queue name
+must be supplied.
+*/
+
+import {
+  ServiceBusClient,
+  SendableMessageInfo,
+  OnMessage,
+  OnError,
+  delay,
+  ReceiveMode,
+  ServiceBusMessage
+} from "../../src";
+
+const connectionString = "";
+const queueName = "";
+
+const testDurationInMilliseconds = 60000 * 10;
```

Comment:
Should this be for 24h?

Reply:
Yes, I initially ran it for 10 min and now it's running for 24 hrs. Can update this.